.circle/index.py

```diff
@@ -67,6 +80,32 @@ def build_index(path_glob, output_path):
     print('Processed %s packs.' % (counter))
     print('Index data written to "%s".' % (output_path))
 
+
+def get_available_versions_for_pack(pack_ref):
+    """
+    Retrieve all the available versions for a particular pack.
+
+    NOTE: This function uses Github API.
+    """
+    url = ('https://api.github.com/repos/%s/%s-%s/tags' %
+           (EXCHANGE_NAME, EXCHANGE_PREFIX, pack_ref))
+    resp = SESSION.get(url)
+
+    if resp.status_code != 200:
+        print('Got non 200 response: %s' % (resp.text))
```

Comment:
Would it be helpful to let the user know what status code was received? All they know here is that it wasn't a 200.

Reply:
I can also include the status code, but the error message itself should already be helpful enough.